.docker/test.ini

```diff
@@ -94,7 +97,7 @@ ckan.redis.url = redis://redis:6379
 #       Add ``datapusher`` to enable DataPusher
 #       Add ``resource_proxy`` to enable resource proxying and get around the
 #       same origin policy
-ckan.plugins = stats text_view image_view recline_view datastore data_qld_theme datarequests data_qld_resources data_qld_integration scheming_datasets validation dcat qa archiver report qgovext data_qld_reporting ytp_comments
+ckan.plugins = stats text_view image_view recline_view datastore data_qld datarequests scheming_datasets validation dcat qa archiver report qgovext ytp_comments
```

Comment:
did we update the order on opsworks cookbook?

Reply:
This is just for CI testing. OpsWorks will still generate the individual plugin names on actual deployments (but all except `data_qld_integration` will point to an empty placeholder plugin).